internal/backends/rlang/rlang.go

```diff
@@ -24,10 +24,8 @@ func createRPkgDir() {
 
 func updateLibPaths() {
 	util.RunCmd([]string{
-		"R",
-		"--no-echo",
-		"-e",
-		`".libPaths('./R/x86_64-pc-linux-gnu-library/3.4')"`,
+		"export",
```

Comment:
I hate to say it, but the `R_LIBS_USER` environment variable will not live past this `RunCmd` call.  If different calls to `R` need `R_LIBS_USER`, you will have to pass it through in the environment.

Reply:
Should be fixed now